pkg/ui/workspaces/db-console/src/views/reports/containers/network/index.tsx

```diff
@@ -113,17 +148,16 @@ export class Network extends React.Component<NetworkProps, INetworkState> {
   refresh(props = this.props) {
     props.refreshLiveness();
     props.refreshNodes();
+    props.refreshConnectivity();
   }
 
   componentDidMount() {
     // Refresh nodes status query when mounting.
     this.refresh();
   }
 
-  componentDidUpdate(prevProps: NetworkProps) {
-    if (!_.isEqual(this.props.location, prevProps.location)) {
-      this.refresh(this.props);
-    }
+  componentDidUpdate(_prevProps: NetworkProps) {
```

Comment:
We should avoid refreshing unconditionally in `componentDidUpdate` moving forward (even if cacheddatareducer will implcity throttle these requests). Is there a better, more reliable way or place in the component to issue this refresh? Since this is already a widely used pattern with these db-console specific apis we can live it here for now if there's no alternative.